config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # Projects
+
+  get "/projects/:id", to: "projects#show"
+  patch "/projects/:id", to: "projects#update"
```

Comment:
See my notes in your controller and view where the form lives. We would want this route to be:
```ruby
post "/projects/:id/contestants", to: "post "/project/:project_id/contestants", to: "project_contestants#create"
```